loadtest.py

```diff
@@ -91,3 +111,15 @@ def test_crash_4mb_uncompressed():
 def test_crash_20mb_uncompressed():
     run_test('test_crash_20mb_uncompressed', 20 * 1024 * 1024,
              compressed=False)
+
+
+@scenario(20)
+def test_crash_400k_uncompressed_multiple_dumps():
+    run_test('test_crash_400k_uncompressed_multiple_dumps', 400 * 1024,
+             dump_names=[
+                 'upload_file_minidump',
+                 'upload_file_minidump_browser',
+                 'upload_file_minidump_flash1',
+                 'upload_file_minidump_flash2',
```

Comment:
I've seen a bunch of dumps with this combination, so seems like a good one to test. There are some other combinations, but Antenna doesn't care about what dumps it's seeing--only how many dumps. Each dump gets sent to s3 in a separate PUT request, so the more dumps there are, the more work Antenna has to do.